### PR TITLE
Remove Elastic Beanstalk configuration

### PR DESCRIPTION
**Pull request by jonnyshaw89**
### What is the context of this PR?
This PR removed the Beanstalk configuration.
Beanstalk is no longer used and ECS is now the preferred way to run survey runner

### How to review
Check that Beanstalk is no longer created
Check that Survey runner still works as expected on ECS

### CODE REVIEW

File: survey-runner.tf [hunk 1]
```diff
@@ -411,40 +327,40 @@ module "survey-register" {
...
-}
+//module "eq-elasticsearch" {
+//  source              = "github.com/ONSdigital/eq-terraform-elasticsearch"
+//  env                 = "${var.env}"
+//  aws_account_id      = "${var.aws_account_id}"
+//  aws_assume_role_arn = "${var.aws_assume_role_arn}"
+//  access_list         = "${concat(module.survey-runner-routing.nat_gateway_ips, split(",", var.ons_access_ips))}"
+//}
+//
+//module "eq-suggest-api" {
```
Comment (bitdivision) [inline survey-runner.tf]: Just delete them? They're in the git history...

**Review by bitdivision — APPROVED** [review]
🎉 🥇 

Tested working.

File: survey-runner.tf [hunk 2]
```diff
@@ -16,46 +16,6 @@ module "eq-alerting" {
...
-  used_jti_claim_table_name        = "${module.survey-runner-dynamodb.used_jti_claim_table_name}"
-  used_jti_claim_dynamo_read       = "${var.survey_runner_used_jti_claim_dynamo_read}"
-  used_jti_claim_dynamo_write      = "${var.survey_runner_used_jti_claim_dynamo_write}"
-  new_relic_enabled                = "${var.survey_runner_new_relic_enabled}"
-  new_relic_app_name               = "${var.env} - ${var.survey_runner_new_relic_app_name} - Beanstalk"
-  new_relic_licence_key            = "${var.survey_runner_new_relic_licence_key}"
-}
-
 module "eq-ecs" {
   source                   = "github.com/ONSdigital/eq-terraform-ecs?ref=v7.0"
```
Comment (samiwel) [inline survey-runner.tf:20]: Do you want to bump this to 7.2 to fix the owners issue in the same PR?

**Review by samiwel — APPROVED** [review]
Seems to work fine. I tested with version 2.0 of the AWS terraform provider so I ran into the owners issue mentioned in my comment below. Fixed by bumping version of eq-terraform-ecs to 7.2

File: survey-runner.tf (hunk 2 above)
Comment (jonnyshaw89) [reply survey-runner.tf:20]: i think that probably just conflates this PR.

**Pull request merged**